js/katex-a11y.js

```diff
@@ -0,0 +1,464 @@
+/**
+ * KaTeX A11y
+ * A library for converting KaTeX math into readable strings.
+ */
+var _ = require("underscore");
+
+if (typeof katex === "undefined" && typeof require !== "undefined") {
+    // NOTE(jeresig): This is mad hacky and used for testing
+    // We want katex to exist in the mocha environment, but we don't
+    // want it to be included in normal builds.
+    var tmp = require;
+    katex = tmp("../docs/js/katex");
+}
+
+var stringMap = {
+    "(": "left parenthesis",
+    ")": "right parenthesis",
+    "[": "open bracket",
+    "]": "close bracket",
+    "\\{": "left brace",
+    "\\}": "right brace",
+    "\\lvert": "open vertical bar",
+    "\\rvert": "close vertical bar",
+    "|": "vertical bar",
+    "\\uparrow": "up arrow",
+    "\\Uparrow": "up arrow",
+    "\\downarrow": "down arrow",
+    "\\Downarrow": "down arrow",
+    "\\updownarrow": "up down arrow",
+    "\\leftarrow": "left arrow",
+    "\\Leftarrow": "left arrow",
+    "\\rightarrow": "right arrow",
+    "\\Rightarrow": "right arrow",
+    "\\langle": "open angle",
+    "\\rangle": "close angle",
+    "\\lfloor": "open floor",
+    "\\rfloor": "close floor",
+    "\\int": "integral",
+    "\\intop": "integral",
+    "\\lim": "limit",
+    "\\ln": "natural log",
+    "\\log": "log",
+    "\\sin": "sine",
+    "\\cos": "cosine",
+    "\\tan": "tangent",
+    "\\cot": "cotangent",
+    "\\sum": "sum",
+    "/": "slash",
+    ",": "comma",
+    ".": "point",
+    "-": "negative",
+    "+": "plus",
+    "~": "tilde",
+    ":": "colon",
+    "?": "question mark",
+    "'": "apostrophe",
+    "\\%": "percent",
+    " ": "space",
+    "\\ ": "space",
+    "\\$": "dollar sign",
+    "\\angle": "angle",
+    "\\degree": "degree",
+    "\\circ": "circle",
+    "\\vec": "vector",
+    "\\triangle": "triangle",
+    "\\pi": "pi",
+    "\\prime": "prime",
+    "\\infty": "infinity",
+    "\\alpha": "alpha",
+    "\\beta": "beta",
+    "\\gamma": "gamma",
+    "\\omega": "omega",
+    "\\theta": "theta",
+    "\\sigma": "sigma",
+    "\\lambda": "lambda",
+    "\\tau": "tau",
+    "\\Delta": "delta",
+    "\\delta": "delta",
+    "\\mu": "mu",
+    "\\rho": "rho",
+    "\\nabla": "del",
+    "\\ell": "ell",
+    "\\ldots": "dots"
+};
+
+var powerMap = {
+    "\\prime": "prime",
+    "\\degree": "degree",
+    "\\circ": "degree"
+};
+
+var openMap = {
+    "|": "open vertical bar",
+    ".": ""
+};
+
+var closeMap = {
+    "|": "close vertical bar",
+    ".": ""
+};
+
+var binMap = {
+    "+": "plus",
+    "-": "minus",
+    "\\pm": "plus minus",
+    "\\cdot": "dot",
+    "*": "times",
+    "/": "divided by",
+    "\\times": "times",
+    "\\div": "divided by",
+    "\\circ": "circle",
+    "\\bullet": "bullet"
+};
+
+var relMap = {
+    "=": "equals",
+    "\\approx": "approximately",
```

Comment:
`\approx` is a verb, so maybe "is approximately" or "approximately equals". Same comment for geq, ge, leq, le, >, <.